.gitignore

```diff
@@ -60,3 +60,4 @@ target
 **/__snapshots__/latest
 errorScreenshots
 themeable-variables.json
+generated-themes
```

Comment:
Should this also be added to the .npmignore and .dockerignore?

Reply:
I agree, it does makes sense to add them there.
Addressed it here : https://github.com/cerner/terra-clinical/pull/599/commits/7501f22e35dc0d4a1a4e6d64155f24dc5818c625